ql/src/test/org/apache/hadoop/hive/ql/metadata/TestSessionHiveMetastoreClientGetPartitionsTempTable.java

```diff
@@ -123,13 +123,13 @@ public void testGetPartitionsByNamesEmptyParts() throws Exception {
     getClient().getPartitionsByNames(DB_NAME, TABLE_NAME, Lists.newArrayList("", ""));
   }
 
-  @Test(expected = MetaException.class)
+  @Test
   @Override
   public void testGetPartitionsByNamesNullDbName() throws Exception {
     super.testGetPartitionsByNamesNullDbName();
   }
 
-  @Test(expected = MetaException.class)
+  @Test
```

Comment:
This is test was introduced to check backward compatibility of the HMS API.
Changing this behavior indicates change in the API. I think we should keep the original Exception.

Reply:
coalesce with above